### PR TITLE
added nats image to image-syncer

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Added NATS image (SHA specific) to image-syncer

**Related issue(s)**

### CODE REVIEW

File: development/image-syncer/external-images.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ images:
 - source: "quay.io/prometheus/prometheus:v2.32.1"
 - source: "quay.io/service-manager/sb-proxy-k8s:v0.9.1"
 - source: "nats@sha256:ae7411548aaccfbe8c77b0f055c26ca31e0f446b32d5e0ac2057d071c679adab"
```
Comment (k15r) [inline development/image-syncer/external-images.yaml:45]: this one is already synced, i guess we can delete it

Comment (mfaizanse) [reply development/image-syncer/external-images.yaml]: done

Comment (mfaizanse) [reply development/image-syncer/external-images.yaml:45]: we cannot change the SHA attached to a tag which is already synced so I added a new tag.

**Review by ammarlakis — APPROVED** [review]

**Pull request merged**